Sources/Overlays/_Testing_CoreGraphics/Attachments/ImageAttachmentError.swift

```diff
@@ -10,8 +10,7 @@
 
 #if SWT_TARGET_OS_APPLE && canImport(CoreGraphics)
 /// A type representing an error that can occur when attaching an image.
-@_spi(ForSwiftTestingOnly)
-public enum ImageAttachmentError: Error, CustomStringConvertible {
+enum ImageAttachmentError: Error, CustomStringConvertible {
```

Comment:
```suggestion
package enum ImageAttachmentError: Error, CustomStringConvertible {
```

Reply:
There's nothing in other targets currently referencing this. Do we actually need it to be `package`?